src/main/resources/templates/categories/dishwashers/dishwashers.ftl

```diff
@@ -15,12 +15,14 @@
       <@govukTextInput.textInput path="form.ecoCapacity"/>
       <@govukTextInput.textInput path="form.energyConsumptionPer100Cycles"/>
       <@govukTextInput.textInput path="form.waterConsumptionPerCycle"/>
-      <div class="govuk-form-group">
-        <@govukFieldset.fieldset legendHeading="Eco programme duration" legendHeadingClass="govuk-fieldset__legend--s" legendSize="h3">
-          <@govukTextInput.textInput path="form.programmeDurationHours"/>
-          <@govukTextInput.textInput path="form.programmeDurationMinutes"/>
-        </@govukFieldset.fieldset>
-      </div>
+      <#if !labelMode?has_content || (labelMode?has_content && labelMode=='LABEL')>
```

Comment:
I think `labelMode` is always set so this doesn't need the `has_content` check. I know we have it in other places, so I'll double check and then update any others in my next commit.